modules/promotion/config/install/core.entity_form_display.node.acm_promotion.default.yml

```diff
@@ -118,19 +119,21 @@ content:
     region: content
     settings: {  }
     third_party_settings: {  }
+  field_coupon_code:
+    weight: 121
+    settings:
+      size: 60
+      placeholder: ''
+    third_party_settings: {  }
+    type: string_textfield
+    region: content
   field_skus:
     weight: 20
     settings:
       size: 60
     third_party_settings: {  }
     type: sku_widget
     region: content
-  langcode:
```

Comment:
@miromichalicka why removing langcode?

Reply:
i ididn't remove it manually, this was exported from latest drupal

Reply:
I wonder if this is going to make the content type not translatable while it should be

Reply:
No, you just won't see language selector while editing

Reply:
pr looks fine to me then